notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-756_analysis.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    data_path = "/home/s/seanmacb/SITCOM/SITCOM-756_local/data/"
</pre><p>I suggest to save in a sub-directory of the notebook directory, not under your home.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>